packages/powersync-db-collection/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@tanstack/powersync-db-collection",
+  "description": "PowerSync collection for TanStack DB",
+  "version": "0.0.0",
+  "dependencies": {
+    "@standard-schema/spec": "^1.0.0",
+    "@tanstack/db": "workspace:*",
+    "@tanstack/store": "^0.7.7",
+    "debug": "^4.4.3",
+    "p-defer": "^4.0.1"
+  },
+  "peerDependencies": {
+    "@powersync/common": "^1.39.0"
+  },
+  "devDependencies": {
+    "@powersync/common": "0.0.0-dev-20251003085035",
```

Comment:
This is currently using a dev version that does not require `Better-SQLite3` or install scripts for downloading the PowerSync Rust core extension. Removing the requirement for install scripts makes running the tests easier.
This dev version requirement is only for unit tests.
This can use the latest version once these have been merged:
- https://github.com/powersync-ja/powersync-js/pull/733
- https://github.com/powersync-ja/powersync-js/pull/690